src/Resources/config/services.yaml

```diff
@@ -0,0 +1,228 @@
+            - '@logger'
+
+    Atoolo\GraphQL\Search\Query\Ping:
+        public: true
+
+    atoolo.search.selectSearcher:
+        class: Atoolo\Search\Service\Search\SolrSelect
+        arguments:
+            - '@atoolo.search.solariumClientFactory'
+            - !tagged_iterator atoolo.search.queryModifier
+            - '@atoolo.search.resultToResourceResolver'
+
+    atoolo.search.suggestSearcher:
+        class: Atoolo\Search\Service\Search\SolrSuggest
+        arguments:
+            - '@atoolo.search.solariumClientFactory'
+
+    atoolo.search.indexer.contentCollector.sitekit:
+      class: Atoolo\Search\Service\Indexer\ContentCollector
+      arguments:
+        - !tagged_iterator atoolo.search.indexer.sitekit.contentMatcher
+
+    Atoolo\Search\Service\Indexer\SiteKit\HeadlineMatcher:
+      tags:
+        - { name: 'atoolo.search.indexer.sitekit.contentMatcher', priority: 10 }
+
+    Atoolo\Search\Service\Indexer\SiteKit\RichtTextMatcher:
+      tags:
+        - { name: 'atoolo.search.indexer.sitekit.contentMatcher', priority: 20 }
+
+    Atoolo\Search\Service\Indexer\SiteKit\QuoteSectionMatcher:
+      tags:
+        - { name: 'atoolo.search.indexer.sitekit.contentMatcher', priority: 20 }
+
+    Atoolo\Search\Service\Indexer\SiteKit\DefaultSchema2xDocumentEnricher:
+        arguments:
+            - '@atoolo.resource.navigationHierarchyLoader'
+            - '@atoolo.search.indexer.contentCollector.sitekit'
+        tags:
+            - { name: 'atoolo.search.indexer.documentEnricher.schema2x', priority: 100 }
+
+    atoolo.search.indexer.translationSplitter:
+      class: Atoolo\Search\Service\Indexer\SiteKit\SubDirTranslationSplitter
+
+    atoolo.search.indexer.aborter:
+        class: Atoolo\Search\Service\Indexer\IndexingAborter
+        arguments:
+            - '%kernel.project_dir%/var/cache/'
+
+    atoolo.search.indexer.locationFinder:
+      class: Atoolo\Search\Service\Indexer\LocationFinder
+      arguments:
+          - '@atoolo.resource.resourceBaseLocator'
+
+    atoolo.search.indexer.statusStore:
+      class: Atoolo\Search\Service\Indexer\IndexerStatusStore
+      arguments:
+        - '%kernel.cache_dir%'
+
+    atoolo.search.indexer.internalResourceIndexerFactory:
+      class: Atoolo\Search\Service\Indexer\InternalResourceIndexerFactory
+      arguments:
+        - !tagged_iterator atoolo.search.indexer.documentEnricher.schema2x
+        - '@atoolo.search.indexer.locationFinder'
+        - '@atoolo.resource.resourceLoader'
+        - '@atoolo.search.indexer.translationSplitter'
+        - '@atoolo.search.indexer.solrIndexService'
+        - '@atoolo.search.indexer.aborter'
+        - 'internal'
+
+    atoolo.search.indexer:
+        class: Atoolo\Search\Service\Indexer\BackgroundIndexer
+        arguments:
+            - '@atoolo.search.indexer.internalResourceIndexerFactory'
+            - '@atoolo.search.indexer.statusStore'
+            - '@logger'
```

Comment:
Yes, I now see it that way too and would make the `atoolo/search`-lib to an `atoolo/search-bundle` so that the bundle can provide these configurations.

Reply:
I assume that means we'll extract it afterwards?

Reply:
I would not separate them. So an `atoolo/search`-Lib and an `atoolo/search-bundle`-bundle but really convert the current project into a bundle. Due to the intensive use of services and the resulting very complex wiring via the configuration, this is necessary in my opinion. I also don't see any meaningful separation of classes that could be split into a lib.

Nothing is extracted, but the configuration remains and is made available via the bundle.

Reply:
I see :+1: